frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -370,36 +368,17 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
 };
 
 export const CSVSubscription: React.FC<CSVSubscriptionProps> = (props) => {
-  type SubscriptionProps = {
```

Comment:
Removes unnecessary `Firehose` when we can just get the data from the parent `DetailsPage` component.